tests/test_plugin_loader.py

```
@@ -1184,6 +1184,215 @@ def test_load_plugins_default_to_main_branch(
 
+    @patch("mmrelay.plugin_loader.clone_or_update_repo")
+    @patch("mmrelay.plugin_loader._install_requirements_for_repo")
+    @patch("mmrelay.plugin_loader.load_plugins_from_directory")
+    @patch("mmrelay.plugin_loader.get_community_plugin_dirs")
+    @patch("mmrelay.plugin_loader.get_custom_plugin_dirs")
+    @patch("mmrelay.plugin_loader.start_global_scheduler")
+    @patch("mmrelay.plugin_loader.logger")
+    def test_load_plugins_missing_ref_warns_unsafe_default(
+        self,
+        mock_logger,
+        _mock_start_scheduler,
```

Ruff (0.15.10)

[warning] Fixture `_mock_start_scheduler` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)